scripts/k8lr

```diff
@@ -1,21 +0,0 @@
-#!/bin/sh
```

Comment:
Shouldn't we keep this file for k8s deployments?

Reply:
can be removed